tutorials/s4sdkjs-deploy-application-cloud-foundry/s4sdkjs-deploy-application-cloud-foundry.md

```diff
@@ -38,9 +30,17 @@ cf login
 
 `cf login` will prompt you for your username and your password. Should you have more then one organization or space, you will also have to select those.
 
-Finally, if you have logged in successfully, you can call `cf push` from the root folder of the project. **`cf` CLI** will automatically pick up the `manifest.yml` of the project.
+Finally, if you have logged in successfully, it is time to build and deploy your application.
+The `package.json` contains a few scripts that can be used for this purpose. In productive environments you would transpile the application from TypeScript to JavaScript using the `ci-build` script, package our deployment using the `ci-package` script and deploy the application using `cf push`.
```

Comment:
As far as I understand, this is not required for the tutorial. Consider making that even more clear, e.g., "For the purpose of the tutorial, this is not required because ... Instead, ..."